assets/js/modules/analytics-4/components/dashboard/ActivationBanner/ReminderBanner.js

```diff
@@ -90,7 +91,19 @@ export default function ReminderBanner( { onSubmitSuccess, children } ) {
 	}
 
 	if ( isLoadingAnalyticsAccess ) {
-		return <ProgressBar />;
+		// Wrap in the googlesitekit-publisher-win class to ensure the ProgressBar is treated in the
+		// same way as a BannerNotification, with only one instance visible on the screen at a time.
+		return (
+			<div className="googlesitekit-publisher-win">
```

Comment:
This immediately struck me as a bit odd—just that we're using this class name for something that isn't _exactly_ a publisher win notification, but looks like we're doing it elsewhere (https://github.com/google/site-kit-wp/blob/694756314dee616db46217e86b084450df9b06ca/assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js#L229-L241), so fair enough 🙂